bolt-sidecar/src/config/mod.rs

```diff
@@ -158,13 +161,17 @@ pub struct Limits {
     /// Maximum number of commitments to accept per block
     pub max_commitments_per_slot: NonZero<usize>,
     pub max_committed_gas_per_slot: NonZero<u64>,
+
+    /// Minimum priority fee to accept for a commitment
+    pub min_priority_fee: NonZero<u64>,
 }
 
 impl Default for Limits {
     fn default() -> Self {
         Self {
             max_commitments_per_slot: NonZero::new(128).expect("Valid non-zero"),
             max_committed_gas_per_slot: NonZero::new(10_000_000).expect("Valid non-zero"),
+            min_priority_fee: NonZero::new(2).expect("Valid non-zero"),
```

Comment:
Can you expose those values in constants? I think it is easier to notice them in that way. Thanks!